application/routes/web.php

```diff
@@ -29,6 +29,8 @@
     Route::middleware('auth:admin')->group(function () {
         Route::post('logout', 'LoginController@logout')->name('logout');
         Route::get('home', 'HomeController@index')->name('home');
+
+        Route::resource('admin_user', 'AdminUserController');
```

Comment:
公式ドキュメントと揃えて
```suggestion
        Route::resource('admin_users', 'AdminUserController');
```
としてください